### PR TITLE
EOS-18144: template files for all provisioner stage

### PR DESCRIPTION
**Pull request by yeshpal-jain-seagate**
- Template files for all stages
- sample template files for CI references

### CODE REVIEW

File: scripts/install/opt/seagate/cortx/motr/conf/motr.reset.tmpl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
```
Comment (huanghua78) [inline scripts/install/opt/seagate/cortx/motr/conf/motr.reset.tmpl:18]: This is an empty template?

Comment (huanghua78) [inline scripts/install/opt/seagate/cortx/motr/conf/motr.reset.tmpl:18]: This is an empty template?

File: scripts/install/opt/seagate/cortx/motr/conf/motr.test.tmpl [hunk 2]
```diff
@@ -0,0 +1,3 @@
+server_node:
```
Comment (huanghua78) [inline scripts/install/opt/seagate/cortx/motr/conf/motr.test.tmpl:1]: This template has no copyright information?

File: scripts/install/opt/seagate/cortx/motr/conf/motr.reset.tmpl (hunk 1 above)
Comment (yeshpal-jain-seagate) [reply scripts/install/opt/seagate/cortx/motr/conf/motr.reset.tmpl]: Removed Empty template files

Comment (yeshpal-jain-seagate) [reply scripts/install/opt/seagate/cortx/motr/conf/motr.reset.tmpl]: Removed Empty template files

**Review by huanghua78 — APPROVED** [review]

**Review by madhavemuri — APPROVED** [review]

**Pull request merged**